examples/with-expo/hooks/useAuth.ts

```diff
@@ -15,6 +15,7 @@ import {
 export default function useAuth() {
   const [loading, setLoading] = useState(true);
   const [user, setUser] = useState<User | null>(null);
+  const [error, setError] = useState(false);
```

Comment:
Interesting approach! I've already seen something like that on UI libraries. The point is that (mainly) this hooks are throwing the errors when they happen, so I'll change just to keep this file using the same standard as the other ones.